assets/css/detours/_detour_table.scss

```diff
@@ -68,6 +68,17 @@ $table-border-radius: 0.5rem;
       }
     }
   }
+
+  &.c-detours-table--active {
+    td {
+      background-color: tokens.$lemon-100;
+      border-color: tokens.$lemon-600 !important;
+    }
+
+    th {
+      border-bottom-color: tokens.$lemon-600 !important;
+    }
+  }
```

Comment:
question(slightly-non-blocking): I was hoping we'd be able to do all of this with bootstrap, are we writing the CSS here because we don't have the bootstrap reboot yet? or because the colors in `ui-alert` don't match "exactly" what the designs used?

Reply:
Yeah let's pair and see what we can find.